apps/src/lib/util/audioApi.js

```diff
@@ -15,99 +17,122 @@ export function injectExecuteCmd(fn) {
   executeCmd = fn;
 }
 
-let audioCommands = {};
-
 /**
- *
- * @param {Object} commands Object holding the set of commands for a droplet
- *        toolkit, which is mutated when passed into this function to add
- *        a set of audio commands.
- * @returns {Object} the mutated commands object
+ * Export a set of native code functions that student code can execute via the
+ * interpreter.
+ * Must be mixed in to the app's command list (see applab/commands.js)
  */
-export function addAudioCommands(commands) {
-  for (const commandName in audioCommands) {
-    if (!audioCommands.hasOwnProperty(commandName)) {
-      continue;
-    }
-
-    if (commands.hasOwnProperty(commandName)) {
-      throw new Error(`Attempted to add "${commandName}" to the command set but it already exists`);
-    }
-
-    commands[commandName] = audioCommands[commandName];
-  }
-  return commands;
-}
+export const commands = {
 
-export function playSound(url) {
-  return executeCmd(null, 'playSound', {'url': url});
-}
+  /**
+   * Start playing a sound.
+   * @param {string} opts.url The sound to play.
+   * TODO:
```

Comment:
Maybe mark w/ username or context of expected implementation (e.g. `will be added as part of ___ feature/implementation`)